airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -221,19 +216,43 @@ def _upload_to_gcs(self, files_to_upload):
         for object, tmp_file_handle in files_to_upload.items():
             hook.upload(self.bucket, object, tmp_file_handle.name, 'application/json')
 
-    @classmethod
-    def convert_types(cls, value):
+    def _convert_types(self, schema, row):
```

Comment:
Make it an instance method because it calls "non-classmethod" `_get_col_type_dict()`. 
`_get_col_type_dict()` cannot be "classmethod" because it uses `self.schema`